app/backend/wells/migrations/0106_auto_20200116_2327.py

```diff
@@ -0,0 +1,52 @@
+# Generated by Django 2.2.9 on 2020-01-16 23:27
+
+from django.db import migrations
+
+
+INSERT_DRILLING_COMPANIES_INTO_ORG = """
+    INSERT INTO registries_organization (create_user, create_date, update_user, update_date, org_guid, name, effective_date, expiry_date, province_state_code)
+        SELECT create_user, create_date, update_user, update_date, drilling_company_guid, name, '2018-01-01 00:00:00', '9999-12-31 00:00:00', 'BC' FROM drilling_company;
```

Comment:
I don't think all companies are in BC, but I also see that we don't have this info for these old records in our own tables.  I'll ask around and maybe we can get it somewhere?

Reply:
Yea they are not (I saw one that has "AB" in the name). When I was chatting with Lindsay — because this is a NOT NULL field — the choices were "BC" or "OTHER". Lindsay suggested that we set them all to "BC" and then later in a different task to manually go through the organizations and clean them up by merging the same ones and updating the province if possible.